homeassistant/components/notify/amazon_chime.py

```diff
@@ -0,0 +1,62 @@
+"""
+Amazon Chime platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.amazon_chime/
+"""
+
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    BaseNotificationService, PLATFORM_SCHEMA)
+from homeassistant.const import CONF_URL
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = []
+_LOGGER = logging.getLogger(__name__)
+
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_URL): cv.url
+})
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the Amazon Chime service."""
+    return AmazonChime(config[CONF_URL])
+
+
+class AmazonChime(BaseNotificationService):
+    """Implement the Amazon Chime Webhook service."""
+
+    def __init__(self, webhook_url):
+        """Initialize the service."""
+        self.webhook_url = webhook_url
+
+    def send_message(self, message='', **kwargs):
+        """Send a message to a Chime webhook."""
+        from requests import post, RequestException
+
+        data = kwargs.get('data', {})
+
+        if data.get('present_members', None):
+            message += ' @Present'
+        if data.get('all_members', None):
+            message += ' @All'
+
+        try:
+            response = post(
```

Comment:
Please move the device interface code, I/O and data parsing, to a standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#6-communication-with-devices-services